MONAIAuto3DSeg/Scripts/auto3dseg_segresnet_inference.py

```diff
@@ -304,7 +304,9 @@ def main(model_file,
 
         # pred = pred.cpu() # convert to CPU if the next step (reverse interpolation) is OOM on GPU
         # invert loading transforms (uncrop, reverse-resample, etc)
-        post_transforms = Compose([Invertd(keys="pred", orig_keys="image", transform=inf_transform, nearest_interp=True)])
+        post_transforms_list = [Invertd(keys="pred", orig_keys="image", transform=inf_transform, nearest_interp=True)]
+        post_transforms_list.append(KeepLargestConnectedComponentd(keys="pred", num_components=2)) if 'whole-head' in model_file else post_transforms_list
```

Comment:
Please keep this inference script model-independent!! I added an issue to track this: https://github.com/lassoan/SlicerMONAIAuto3DSeg/issues/69